### PR TITLE
Feature/update edit profile form

### LINKED ISSUE #914: Create user profile template
When migrating the old frontend templates to Django, we noticed the user profile was not rendering any content. Rather than weed through the proclaimed "horrendous code" that was responsible for rendering the BackBone view, let's just write a _simple_ user profile template from scratch. 

https://github.com/CiviWiki/OpenCiviWiki/blob/374381b6e4b9b556580157f33fbd8c67204c1b36/project/webapp/static/js/account.js#L32-L33

## Task
- determine what fields are available on the user profile
- sketch a basic user profile page using pencil and paper or a drawing app
- discuss the proposed sketch
- implement the sketch using conventional Django elements like Form and class-based views

### PR DESCRIPTION
**Pull request by wassafshahzad**
What does this PR do ?

- This PR removes The Edit profile form and renders a read only settings forms.
- This PR fixes an issue where a new user is unable to setup  account.
- This PR makes the setting form reusable.

What is  the relevant tickets:
Closes #914 

Images
Profile

### CODE REVIEW

**Comment by alanCalvillo-Wizeline** [comment]
The code looks good, is it possible to have some sort of basic unit testing for this new update that you are including in the project ?

**Comment by wassafshahzad** [comment]
> The code looks good, is it possible to have some sort of basic unit testing for this new update that you are including in the project ?

Yeah i could add test for the forms and the view . if that is ok

**Comment by brylie** [comment]
@all-contributors please add @alanCalvillo-Wizeline for review

**Review by brylie — CHANGES_REQUESTED** [review]
As per @alanCalvillo-Wizeline's feedback

> The code looks good, is it possible to have some sort of basic unit testing for this new update that you are including in the project ?

File: project/accounts/templates/accounts/utils/update_settings_form.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<div class="step-title">
+    <div class="section">
+        <div class="title-lato center">
+            Account Settings
```
Comment (brylie) [inline project/accounts/templates/accounts/utils/update_settings_form.html:4]:
```suggestion
            Profile
```

**Review by brylie — APPROVED** [review]

File: project/threads/templates/threads/partials/account/account_base.html [hunk 2]
```diff
@@ -14,7 +14,7 @@
                             <li class="tab"><a href="#issues">MY ISSUES</a></li>
                             {% if request.user.username == username|stringformat:"s" %}
                             <li class="tab">
-                                <a href="#settings">EDIT PROFILE</a>
+                                <a href="#settings">SETTINGS</a>
```
Comment (brylie) [inline project/threads/templates/threads/partials/account/account_base.html]:
```suggestion
                                <a href="#settings">Settings</a>
```

**Review by brylie — APPROVED** [review]

**Pull request merged**